test/integration/template_backup_test.go

```diff
@@ -38,7 +38,7 @@ func TestDatabaseBackupCronJobDisabled(t *testing.T) {
 	// Run RenderTemplate to render the template and capture the output.
 	_, err := helm.RenderTemplateE(t, options, helmChartPath, "release-name", []string{"templates/cronjob.yaml"})
 
-	assert.Contains(t, err, "could not find template")
+	assert.Contains(t, err.Error(), "could not find template")
```

Comment:
should this just be `assert.ErrorContains`?

Reply:
There is no `ErrorContains` in github.com/stretchr/testify/assert. But I needed it for `Subset` function.

Reply:
I see that you changed gotest to stretcher. Got it!

I wonder why it was left behind...

Reply:
could you remove it from `go.mod` and `go.sum` so that people don't use it accidentally?

Reply:
Good point! I will remove `gotest.tools` ... it looks like it isn't used anywhere else.